lib/components.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
```

Comment:
I like that you created a components file, however, I think it would be better to consider other components. So could you make components a folder and create a file inside called offTopTitle.dart

Reply:
🤦‍♂ rookie mistake